### PR TITLE
AIFBR-12442 : Delete OOB package version

### PR DESCRIPTION
**Pull request by vaibhavs-uipath**

### CODE REVIEW

File: mlpackage/deleteoob/input.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+	"dbConnection": "sfdev3014441-a0d04dcc-sql.database.windows.net",
```
Comment (vivekjhauipath) [inline mlpackage/deleteoob/input.json:2]: remove these values, instead put <<db connection string>> etc.

File: mlpackage/deleteoob/deleteoobpackageversion.sh [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+  ml_package_version_id=${ml_package_version_details[0]}
+  echo ml_package_version_id
+  account_id=${ml_package_version_details[1]}
+  echo $account_id
+  tenantId=${ml_package_version_details[2]}
+  echo $tenantId
+  projectId=${ml_package_version_details[3]}
+  echo $projectId
+  contentUri=${ml_package_version_details[4]}
+  echo $contentUri
```
Comment (vivekjhauipath) [inline mlpackage/deleteoob/deleteoobpackageversion.sh:101]: nit: with dockerImage support this field can be null, in which case just delete entry and no zip file

**Review by vivekjhauipath — CHANGES_REQUESTED** [review]
1. Remove creds etc from commit. Squash so that it doesn't appear in history.
2. Delete package if no more versions present. Else bad UX
3. Check with PMs if we need to delete cloned versions based on this version too? Could be an enhancement

**Comment by vaibhavs-uipath** [comment]
> 1. Remove creds etc from commit. Squash so that it doesn't appear in history.
> 2. Delete package if no more versions present. Else bad UX
> 3. Check with PMs if we need to delete cloned versions based on this version too? Could be an enhancement

@vivekjhauipath  These are done,please check.

**Review by pramod-rao-uipath — APPROVED** [review]
LGTM

**Review by vnaren23 — APPROVED** [review]

**Review by vivekjhauipath — APPROVED** [review]
LGTM.
Though we aren't handling cloned version cascading delete, but we can probably do that from UI, so should be fine.

**Comment by vivekjhauipath** [comment]
> > 1. Remove creds etc from commit. Squash so that it doesn't appear in history.
> > 2. Delete package if no more versions present. Else bad UX
> > 3. Check with PMs if we need to delete cloned versions based on this version too? Could be an enhancement
> 
> @vivekjhauipath These are done,please check.

3rd is not done.

**Comment by vivekjhauipath** [comment]
Also, @vaibhavs-uipath minor enhancement could be to support customVersion also.

**Pull request merged**